### PR TITLE
Portfolio contact section

### PR DESCRIPTION
**Pull request by ezraermy**
## Portfolio Contact Section
This is a pull request. I:
- created branches with different base branches.
- Followed GitHub flow.
- Added a descriptive and easy to understand README
- Used a HTML 
- Used a CSS
- Added contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,151 +1,210 @@
...
-</html>
\ No newline at end of file
+    <footer class="contact">
+      <h1 class="contact-title">Contact me</h1>
+      <p class="contact-content">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="" method="" class="my-form">
```
Comment (cynthiainga) [inline index.html]: - [ ] As required in the project requirements the form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
  - `action` value should be similar to `https://formspree.io/f/{form_id}`
  - `method` value equals `POST`
Please kindly fix that 🙃 🙃

File: style.css [hunk 2]
```diff
@@ -370,3 +342,76 @@ div .taggs {
...
+
+.contact {
+  width: 98%;
+  background: #6070ff;
+  background-image: url(images/contact/layers.svg);
+  background-repeat: no-repeat;
+  background-position-x: 135px;
+  background-position-y: 10px;
+  border-top-left-radius: 6rem;
+}
```
Comment (cynthiainga) [inline style.css]: - [ ] Please kindly make sure your design stick as much as possible to the Figma design. Your background image is not aligned properly, kindly fix their position to the right 🙃 😉

Your design

Screen Shot 2022-11-02 at 00 00 42

Figma design

Screen Shot 2022-11-02 at 00 01 32

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @ezraermy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -1,151 +1,211 @@
...
-</html>
\ No newline at end of file
+    <footer class="contact">
+      <h1 class="contact-title">Contact me</h1>
+      <p class="contact-content">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/{form_id}" method="POST" class="my-form">
```
Comment (ridwanediallo) [inline index.html]: - [ ] Hello @ezraermy, good job so far 👏. But the form submission is still not working.
 
Screenshot from 2022-11-01 23-09-05

- [x] What you need first is to create your account at [formspree](https://formspree.io/html/) and then generate the `form_id` by following the steps. 
- [x] In your `action` value replace `{form_id}` with the `id` you generated from  [formspree](https://formspree.io/html/). 👌
when you got the `id`, your `action` should be simmilar to something like this `action="https://formspree.io/f/hgkd1254us"`
- [x] Lastly, please make that a user can successfully submit a `form`. After that is done you can proudly submit it for review. 🤗

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @ezraermy , 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (ridwanediallo) [reply index.html]: ## How to get the `action` value ?
[Demo video](https://user-images.githubusercontent.com/89228743/199359877-b51846d3-16b3-488d-92c2-a992ecee2b21.webm)

**Review by emmiiorji — APPROVED** [review]
Hi @ezraermy, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Red-approved

## Good Highlights

- Linter checks are passing
- Adherence to GitHub flow
- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**